npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -159,6 +159,8 @@ def path_dependencies(fetched_files)
 
         path_dependency_details(fetched_files).each do |name, path|
           path = path.gsub(PATH_DEPENDENCY_CLEAN_REGEX, "")
+          raise PathDependenciesNotReachable, name if path.start_with?("/")
```

Comment:
What do you think of leaving a little more evidence for debugging?

```suggestion
          raise PathDependenciesNotReachable, "#{name} at #{path}" if path.start_with?("/")
```